test/export/StructureDefinitionExporter.test.ts

```diff
@@ -21,6 +21,8 @@ import {
 import { loggerSpy, TestFisher } from '../testhelpers';
 import { ElementDefinitionType } from '../../src/fhirtypes';
 import path from 'path';
+import { logger } from '../../src/utils';
```

Comment:
As github actions has pointed out 'logger' is defined but used. Could you remove that.

Reply:
Hey... so I have a follow-on PR to this for the IG dependencies and I actually already found that issue and fixed it in a commit on my other branch.  Can you trust me to fix it in my next PR, which will be ready 2 minutes after we merge this?

Reply:
Alright @cmoesel , if you say so. :) 💯